src/index.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const { colors } = require("@primer/primitives");
+
+const getTheme = require("./theme");
```

Comment:
The theme shall reside in this one single place and it references the Primer primitives rather than hardcoding any colors.